LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -750,6 +750,13 @@ def handle_config(config_data, me_url, mdsd_url, is_lad):
     if "virtualMachineScaleSets" in az_resource_id: 
```

Comment:
I think this comment needs to be changed, it gives impression it is about vmss id, but should have been vmss instance resource id or examples are wrong:
...... If the instance is VMSS then trim the last two values from the resource id ie - "/virtualMachines/0"
   Since ME expects the resource id in a particular format. For egs -
    IMDS returned ID - /subscriptions/<sub-id>/resourceGroups/<rg_name>/providers/Microsoft.Compute/virtualMachineScaleSets/<VMSSName>/virtualMachines/0
  ME expected ID- /subscriptions/<sub-id>/resourceGroups/<rg_name>/providers/Microsoft.Compute/virtualMachineScaleSets/<VMSSName>......................
    
 VMSS resource ID only is formatted to contain VMSS name, when VMSS instance resource ID is the VMSS child instance ID, which has /0/ ending depending on the instance name. Is ME only taking VMSS ID ? if yes, why we require setting for LAD to point to VMSS instance resource ID which contains /0/ at the end and not directly VMSS ID?

Reply:
updated the comments